WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -466,15 +466,8 @@ class PageListViewController: AbstractPostListViewController, UIViewControllerRe
     private func show(_ editorViewController: EditorViewController) {
         let filterIndex = filterSettings.currentFilterIndex()
 
-        editorViewController.onClose = { [weak self, weak editorViewController] changesSaved, _ in
-            if changesSaved {
-                if let postStatus = editorViewController?.post.status {
-                    self?.updateFilterWithPostStatus(postStatus)
-                }
-            } else {
-                self?.updateFilter(index: filterIndex)
-            }
-
+        editorViewController.onClose = { [weak self, weak editorViewController] _, _ in
+            self?.updateFilter(index: filterIndex)
```

Comment:
```suggestion

```

It looks like we do not need to keep this at all if we also remove the `updateFilterWithPostStatus` call on line 480 which moves the tab to Draft right away. 

https://github.com/wordpress-mobile/WordPress-iOS/blob/67cc42ce581751fb52f16f710a59e41e00aa31aa/WordPress/Classes/ViewRelated/Pages/PageListViewController.swift#L480